### ISSUE TITLE
Failed to collect information from stac-download. 'req_progress' is not an exported object from 'namespace:httr2'

### ISSUE THREAD

**Issue by sermomon** [pepijn-devries/CopernicusMarine#46]
I am trying to run the tutorial shown on the home page. Due to the problems mentioned in #42 I am trying to use STAC for downloading a complete set. I am doing the following:

```
options(CopernicusMarine_uid = "sermomon")
options(CopernicusMarine_pwd = "**********")
```

```
stac_files <-
  cms_list_stac_files(
    "GLOBAL_ANALYSISFORECAST_PHY_001_024",
    "cmems_mod_glo_phy-cur_anfc_0.083deg_P1D-m")
stac_files
```

```
# A tibble: 842 × 8
   home  native current_path LastModified ETag  Size  StorageClass Type 
   <chr> <chr>  <chr>        <chr>        <chr> <chr> <chr>        <chr>
 1 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"a… 1938… STANDARD     Norm…
 2 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"d… 1938… STANDARD     Norm…
 3 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"9… 1938… STANDARD     Norm…
 4 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"e… 1938… STANDARD     Norm…
 5 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"e… 1938… STANDARD     Norm…
 6 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"f… 1938… STANDARD     Norm…
 7 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"6… 1938… STANDARD     Norm…
 8 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"2… 1938… STANDARD     Norm…
 9 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"c… 1938… STANDARD     Norm…
10 s3.w… mdl-n… native/GLOB… 2023-11-17T… "\"a… 1938… STANDARD     Norm…
# ℹ 832 more rows
# ℹ Use `print(n = ...)` to see more rows
```

I get the list of files properly, however when I try to download the first element I get an error.

`cms_download_stac(stac_files[1,,drop = FALSE], tempdir())`

Error:

```
Failed to collect information from stac-download.
'req_progress' is not an exported object from 'namespace:httr2'
```

I don't know if it is a package issue, some third party issue or some error due to my configuration. In this case I apologize in advance and thank you for any help.

**Comment by sermomon** [comment]
Solved updating https2 to ‘1.0.1’.

**Comment by sermomon** [comment]
Solved updating https2 to ‘1.0.1’.

**Comment by pepijn-devries** [comment]
Thanks for reporting and figuring out a fix. I will have a look at adding minimum version numbers for the package dependencies.
